src/control/server/init/setup_spdk.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
Why make this change? Personally I prefer `/usr/bin/env bash` in case someone has something special in their env

Reply:
for bash?

I found this frustrating, the python code is IMO correct where it calls "env python" and that's a requirement to use any non-trivial python libraries.

However - our spec file was replacing all env scripts by replacing "env " with "" so both this and the python scripts were in the rpm as using /usr/bin/python or /usr/bin/bash but then also there's a step which I think is the rpm install process itself where the python scripts get re-written a second time so after you install the rpms the python code is installed with /usr/bin/python3.6, at which point I gave up and changed the invocation to be "python launch.py" - I think my conclusion is rpm just doesnt' want you to be able to use non-rpm software.

This PR removes that modification that is done in our spec file and adds exceptions to silence the rpmlint warnings but it's pointless anyway as the code gets re-written a second time at install.

Then - just to top it off our ubuntu node deployment re-writes it a third time to use "python2".

Reply:
That's messy... I defer to you :)